modules/core/src/passes/layers-pass.js

```diff
@@ -19,7 +19,8 @@ export default class LayersPass extends Pass {
     pass = 'draw',
     redrawReason = '',
     customRender,
-    effects
+    effects,
```

Comment:
Perhaps just pass down the `effectManager` depending on how we resolve previous comment.